#Array of detected object
Detection[] detected_object